tests/Functional/Schema/AlterTableTest.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Schema;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Platforms\AbstractMySQLPlatform;
+use Doctrine\DBAL\Platforms\DB2Platform;
+use Doctrine\DBAL\Platforms\OraclePlatform;
+use Doctrine\DBAL\Platforms\SQLitePlatform;
+use Doctrine\DBAL\Platforms\SQLServerPlatform;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Types\Types;
+
+class AlterTableTest extends FunctionalTestCase
+{
+    /** @throws Exception */
```

Comment:
What's the point of this annotation?

Reply:
`@throws` annotations on unit tests don't make much sense. Noboby's going to catch them except for PHPUnit itself. We should remove them.

Reply:
Yes… maybe this could even be reported to PHPStorm, I guess they must have some kind of PHPUnit integration.

Reply:
Reported as [WI-80744](https://youtrack.jetbrains.com/issue/WI-80744).